grakn-client/src/main/java/ai/grakn/remote/concept/RemoteAttributeType.java

```diff
@@ -65,4 +66,14 @@
     public final String getRegex() {
         return getProperty(ConceptProperty.REGEX);
     }
+
+    @Override
+    final AttributeType<D> asSelf(Concept concept) {
```

Comment:
because `from` is a bad word :P

Reply:
I love this method btw. It's like your `getThis()` but also:
1. Type-safe! No way it can crash.
2. Because it accepts a `Concept`, you can use it to cast the result of `sup()` or `subs()`.